Makefile.cache

```diff
@@ -169,18 +170,22 @@ endef
 define GET_MOD_DEP_SHA
 	$(eval $(1)_MOD_DEP_PKGS := $(foreach dfile,$($(1)_DEPENDS) $($(1)_RDEPENDS) $($(1)_WHEEL_DEPENDS) \
                             $($(1)_PYTHON_DEBS) $($(1)_PYTHON_WHEELS) \
-                            $($(1)_DBG_DEPENDS) $($(1)_DBG_IMAGE_PACKAGES) $($(1)_LOAD_DOCKERS),\
```

Comment:
Why remove _DBG_IMAGE_PACKAGES?

Reply:
DBG_IMAGE_PACKAGES is removed because this contains names of debain packages  that get installed through apt-get command and this is not a real file for GIT SHA calculation.